src/modules/pass.js

```diff
@@ -227,11 +227,12 @@ PassFF.Pass = (function () {
         }
         return browser.runtime.sendNativeMessage("passff", args)
           .then((result) => {
-            let version = result.version || "0.0";
-            if (version !== "1.0.1" && version !== "testing") {
+            let version = String(result.version) || "0.0";
```

Comment:
This will never fall back to "0.0", but you might end up with version strings like "undefined" or "null". How about `String(result.version || "0.0")`?